src/components/ImageView/index.js

```diff
@@ -61,6 +61,11 @@ class ImageView extends PureComponent {
         document.removeEventListener('mousemove', this.trackMovement.bind(this));
     }
 
+    /**
+     * handle on image container layout change
```

Comment:
This comment should be removed
https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs